hpat/hiframes/pd_series_ext.py

```diff
@@ -683,14 +683,6 @@ def resolve_head(self, ary, args, kws):
         assert not kws
         return signature(ary, *args)
 
-    @bound_function("series.median")
-    def resolve_median(self, ary, args, kws):
-        assert not kws
-        dtype = ary.dtype
-        # median converts integer output to float
-        dtype = types.float64 if isinstance(dtype, types.Integer) else dtype
-        return signature(dtype, *args)
-
```

Comment:
do not delete this. Comment it out if you need. In this PR this code is needed for parallelism.

Reply:
@shssf But that's just typing the same we've got from the overload. Keeping it has no sense - if we don't use old-style implementation from series_kernels.py, i.e. hpat.hiframes.api.median.